packages/office-addin-debugging/src/cli.ts

```diff
@@ -19,12 +19,12 @@ commander
     .option("--document <document>", "Document to be used for sideloading.")
     .option("--no-debug", "Start without debugging.")
     .option("--no-live-reload", "Do not enable live-reload.")
-    .option("--dev-tools", "Open the web browser developer tools when debugging (if supported)")
+    .option("--no-sideload", "Do not start the Office app and load the add-in.")
```

Comment:
Shouldn't it just say something like: "Start the Office app, but do not load the add-in"?
The way that is currently written makes it sounds like nothing opens (neither the app nor the add-in)

Reply:
should we add this to the ReadMe

Reply:
Added

Reply:
@igor-ribeiiro It is written that way because the concept of sideloading covers both launching the app and loading the add-in. If sideload is turned off it will not launch the app when you start debugging.